src/main/java/com/linkedin/kmf/apps/MirrorPipelineMonitor.java

```diff
@@ -70,6 +70,7 @@ public MirrorPipelineMonitor(Map<String, Object> props, String name) throws Exce
                                                                      String topic,
                                                                      String name) throws Exception {
     Map<String, Object> topicManagementProps = createTopicManagementServiceProps(props, topic);
+    topicManagementProps.put(CommonServiceConfig.ZOOKEEPER_CONNECT_CONFIG, "");
```

Comment:
Yeah I get it is required config. But why do we provide an empty string here? Since it is required config, shouldn't it already be specified in the `props` which is passed it `MirrorPipelineMonitor`?